src/apis/axiosClient.js

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+
+const axiosInstance = axios.create({
+  baseURL: 'http://localhost:8000/api/v1/auth',
```

Comment:
```suggestion
  baseURL: 'http://localhost:8000/api/v1',
```

We will be making api calls to other endpoints as well like /api/v1/users, so we don't want all of them to start with `auth`

Reply:
Eg: take a look at the endpoints that have been documented so far: https://musicstore.onrender.com/api-docs/

Reply:
I removed `auth` from url